src/components/pageComponent/login/loginBox.tsx

```diff
@@ -0,0 +1,75 @@
+import ButtonCta from "../../common/buttonCta/buttonCta";
+import Input from "../../common/input/input";
+import style from "./loginBox.module.scss";
+import TopMenuLogin from "./topMenuLogin";
+import {ChangeEvent, FormEvent, useState} from "react";
+import {createLoginToken} from "../../../data/data-global.jsx";
+import Label from "../../common/input/label";
+interface LoginValues {
+  email: string;
+  password: string;
+}
+function LoginBox() {
+  const [loginValues, setLoginValue] = useState<LoginValues>({
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const {name, value} = e.target;
+    setLoginValue((prevLoginValues) => ({
+      ...prevLoginValues, // 이전 복사본을 불러온다
+      [name]: value, // 복사본을 수정한다
+    }));
+  };
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    //const formData = new FormData();
+    //formData.append("email", loginValues.email);
+    //formData.append("password", loginValues.password);
+    createLoginToken(loginValues);
+  };
+  return (
+    <form method="post" className={style.loginBox} onSubmit={handleSubmit}>
+      <TopMenuLogin />
+      <div className={style.inputBox}>
+        <Label htmlFor="email" text="이메일" />
+        <Input
+          id="email"
+          name="email"
+          type="text"
+          value={loginValues.email}
+          placeholder="아이디 입력"
+          onChange={handleChange}
+        />
+        <Label htmlFor="password" text="비밀번호" />
+        <Input
+          id="password"
+          name="password"
+          type="password"
+          value={loginValues.password}
+          placeholder="비밀번호 입력"
+          onChange={handleChange}
+          icon={true}
```

Comment:
컴포넌트의 props로 boolean값을 넘길때, true인 경우에는 이렇게 표현해주는 편입니다~
```suggestion
          icon
```